src/java/org/apache/cassandra/index/sai/disk/v1/MemtableIndexWriter.java

```diff
@@ -89,7 +89,12 @@ public void addRow(PrimaryKey key, Row row, long sstableRowId)
     @Override
     public void abort(Throwable cause)
     {
-        logger.warn(indexIdentifier.logMessage("Aborting index memtable flush for {}..."), indexDescriptor.sstableDescriptor, cause);
+        if (cause == null)
+            // This commonly occurs when a Memtable has no rows to flush, and is harmless:
+            logger.debug(indexIdentifier.logMessage("Aborting index memtable flush for {}..."), indexDescriptor.sstableDescriptor);
+        else
+            logger.warn(indexIdentifier.logMessage("Aborting index memtable flush for {}..."), indexDescriptor.sstableDescriptor, cause);
+
```

Comment:
@mike-tr-adamson @adelapena This makes some of the tests noisier, given they don't write tons of data or write data to all columns in all cases...so I just fixed it.